src/modules/pages/EditPage/editApiContainer.js

```diff
@@ -2,22 +2,34 @@ import { connect } from 'react-redux';
 
 import {
     deleteEndpoint,
+    excludePlugin,
+    fillSelected,
+    selectPlugin,
+    fetchEndpointSchema,
     fetchEndpoint,
     refreshEndpoints,
     resetEndpoint,
+    saveEndpoint,
     updateEndpoint,
 } from '../../../store/actions';
 
 import ApiItem from './ApiItem';
 
 const mapStateToProps = state => ({
     api: state.apiReducer.api,
+    selectedPlugins: state.apiReducer.selectedPlugins,
+    // selectedPlugins: state.apiReducer.api.plugins.map(item => item.name),
```

Comment:
I'd remove commented code

Reply:
this is my personal **Dragon** I'm fighting with 🐉